src/image/imageengine.js

```diff
@@ -54,6 +55,20 @@ export default class ImageEngine extends Plugin {
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'src' );
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'alt' );
 
+		// Responsive sizes default value. More information and discussion on this topic:
+		// https://github.com/ckeditor/ckeditor5-image/issues/2.
+		config.define( 'image.responsiveSizes', '100vw' );
```

Comment:
Ok, so I misunderstood. I thought that we are not going with config option to insert  `[RESPONSIVE_IMAGE_SIZES]` as `sizes` value but staying with possibility to change default value.